test/CMakeLists.txt

```diff
@@ -275,7 +269,7 @@ endif (PYTHON_BINDINGS)
 function(add_depict_test testname obfile image_file ref_image)
   set(depict_dir ${CMAKE_BINARY_DIR}/depict_test)
   add_test(NAME test_${testname} COMMAND ${CMAKE_SOURCE_DIR}/test/test_depiction.sh
-      ${CMAKE_BINARY_DIR}/bin/obabel ${ImageMagick_convert_EXECUTABLE}
+      ${CMAKE_BINARY_DIR}/bin/babel ${ImageMagick_convert_EXECUTABLE}
```

Comment:
Should not be modified.

Reply:
You are right. This file should be fixed.